src/main/java/co/cask/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -70,6 +70,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     String projectId = config.getProject();
     properties.put("fs.gs.project.id", projectId);
     properties.put("fs.gs.system.bucket", GCSConfigHelper.getBucket(config.path));
+    properties.put("fs.gs.working.dir", GCSConfigHelper.getBucket(config.path));
```

Comment:
the warning message says it defaults to gs://bucket/, which is different than this. Have we tested this to make sure it works?

In any case, I think it's safer to set it to gs://bucket/

Reply:
Yea, I did test these changes. The output path was same as provided in the plugin config and there were no warn messages in the pipeline. However the default value is root of the filesystem. https://github.com/GoogleCloudPlatform/bigdata-interop/blob/v1.9.1/gcs/conf/gcs-core-default.xml#L13

So I have changed it to be `/` which is relative to the `fs.gs.system.bucket` 

**Output Path** `gs://bq_cdap_batch_sink/path4/blah`

![image](https://user-images.githubusercontent.com/14131070/47462693-ff1ea780-d798-11e8-870f-d98c5af1460e.png)

**Pipeline log levels:**
![image](https://user-images.githubusercontent.com/14131070/47462735-1c537600-d799-11e8-82dc-0e03291b8340.png)